src/main/java/jnr/constants/platform/fake/LastError.java

```diff
@@ -0,0 +1,157 @@
+// WARNING: This file is autogenerated. DO NOT EDIT!
```

Comment:
should it be here or not? it wasn't generated before

Reply:
Since it's generated now, I suppose it should be there...